### PR TITLE
Pylint-suggested fixes in GPhL code - and a few obvious errors

### PR DESCRIPTION
**Pull request by rhfogh**

### CODE REVIEW

File: HardwareObjects/mockup/CollectEmulator.py [hunk 1]
```diff
@@ -26,12 +26,12 @@
 import subprocess
 import logging
 import re
+from collections import OrderedDict
 import f90nml
+import api
```
Comment (marcus-oscarsson) [inline HardwareObjects/mockup/CollectEmulator.py]: So is this the Beamline object we discussed or the API ?

Comment (rhfogh) [reply HardwareObjects/mockup/CollectEmulator.py]: Dear MArcus,

No, this is Ivars api object. It should be changed over to the one we discussed, assuming that we do agree on that, of course.

Comment (marcus-oscarsson) [reply HardwareObjects/mockup/CollectEmulator.py]: Okey, so you will do that in another PR then I imagine ?

Comment (rhfogh) [reply HardwareObjects/mockup/CollectEmulator.py]: Yes

File: HardwareObjects/BlissRontecMCA.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 from HardwareRepository.BaseHardwareObjects import HardwareObject
-from AbstractMCA import *
+from abstract.AbstractMCA import AbstractMCA
```
[the remark this answers is not in the document]
Comment (rhfogh) [reply HardwareObjects/BlissRontecMCA.py]: Indeed. Fixed now.

**Pull request merged**